tests/unit/test_models.py

```diff
@@ -49,6 +50,8 @@ def test_models(tmpdir: "Path"):
     ) as testmodel:
         testmodel1tag = testmodel.tag
 
+    time.sleep(1)
```

Comment:
maybe we can check whether this is running in actions
```suggestion
if os.getenv("GITHUB_ACTIONS"):
   ...
```
So that when test run locally we don't have to sleep it

Reply:
Please address address @aarnphm's suggestion in a following PR if needed

Reply:
I think we shouldn't special case our tests too much, it'll just make debugging harder.

Though I should probably debug what's going on here at some point...